data/remnant/remnant outfits.txt

```diff
@@ -474,6 +474,19 @@ outfit "Emergency Ramscoop"
 	"ramscoop" 0.5
 	description "The Ember Waste is the sort of place where only a very foolish captain would travel without some sort of device for replenishing hyperspace fuel. While this risk is non-existent in Remnant ships due to their 'gills', which are effectively built-in ramscoops, some captains prefer to speed up the process by adding additional capacity. For the rare captain braving the area in non-Remnant ships, this ramscoop is not particularly powerful, but it allows a ship to explore without fear of getting stranded."
 
+outfit "NDR-114 Android"
+	category "Systems"
```

Comment:
We have quite a few outfits now that we gift to the player that they can never reacquire, Nanachi's pendant and the Outskirts Gauger to name two others. Should we make a new category titled "Unique" to denote these outfits as something to keep an eye on? We could even make it so that "Unique" outfits are harder to lose track of; perhaps when you sell a ship with "Unique" outfits on it, the outfits go to storage instead of being sold alongside the ship, or you get an extra pop-up warning you that you're selling a ship with unique outfits. This same pop-up could appear if you try to sell the outfit directly.

Just some food for thought, not something I think needs done in this PR.

Reply:
I like that idea. Is there any reason I can't just change the category here now to start the process? 
I can do a separate PR to get the other unique things to keep this PR focused/clean; but giving said category any special consideration regarding preservation or whatnot would need code. 

One "simple" idea would be that when a ship is sold, anything in the "unique" category that is onboard is automatically uninsntalled into storage instead of actually sold. Likewise, even if it is cargo, it is never sold from the outfitter either. If the player actually wants to sell unique things, they have to actually go in and manually sell the outfit. Could go with lore/tooltip that the general shipyard people don't know what to do with truly unique things, so they just set it aside for the player's decision; but someone will buy it from them if the player explicitly sells it to them.

Reply:
I have changed them to "Unique" and added the category.

It occurs to me, though, that perhaps this is going about it the wrong way. "Special" is close enough to "Unique" that it makes sense for the various unique things to be considered special. The real grouping is probably pulling all the licenses *out* of "special" and putting them in their own "Licenses" category. Or perhaps just renaming the "Special" category to "Licenses" and pulling the rest of the stuff out of it into a new special or unique category.